File: Android/Vine-5.50.1/Decompilation/app/src/main/java/co/vine/android/scribe/model/ActivityDetails.java
package co.vine.android.scribe.model;

import com.bluelinelabs.logansquare.annotation.JsonField;
import com.bluelinelabs.logansquare.annotation.JsonObject;

@JsonObject
/* loaded from: classes.dex */
public class ActivityDetails {

    @JsonField(name = {"activity_id"})
    public Long activityId;

    @JsonField(name = {"activity_type"})
    public String activityType;

    @JsonField(name = {"n_more"})
    public Integer nMore;
}

File: Windows Phone 8.1/2.5.7.0/Decompilation/Vine.WindowsPhone/Vine.Models.Analytics/ActivityDetails.cs
using Newtonsoft.Json;

namespace Vine.Models.Analytics;

public class ActivityDetails
{
	[JsonProperty("activity_id")]
	public long ActivityId { get; set; }

	[JsonProperty("activity_type")]
	public string ActivityType { get; set; }

	[JsonProperty("n_more")]
	public int NMore { get; set; }
}
